cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -164,202 +163,197 @@ protected TimeSeries getTimeseries(String page, int pageSize, String names, Stri
         final String recordCursor = cursor;
         final int recordPageSize = pageSize;
 
-        try {
```

Comment:
This mostly looks like it was just formatting tweaks. Was the performance of this endpoint verified after all these changes in any way?

Reply:
I should have commented on these changes.  In getTimeseries() there was a try/catch that adapted DataAccessException into a NotFoundException - that now happens in the listener for almost all jooq.  Removing the try/catch change the indent for a bunch of code so it looks like a huge change but it really isn't.  
The create/store/delete previously used dsl.connection(())  - that was changed to use the new JooqDao.connection(dsl, ()) method so that exceptions from create/store/delete will be adapted in a consistent way.

Reply:
I manually did a TS retrieve after these changes but I didn't try out all the other methods or the failure cases.

Reply:
That explanation works for me. Always a bit tricky when so much code is getting shifted.